go/vt/automation/testing_emit_echo_task.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2015, Google Inc. All rights reserved.
```

Comment:
I think we use register pattern a lot. Basically, you could add a Register(name string, taskCreator func() Task) function, so that the test task could call the Register func in its init func. Therefore, the createTaskInstance func will look like getRegisteredTaskCreator(name)(), which you get a registered task creator and then create a task. This is my suggestion.